superset/assets/src/components/Asterisk.jsx

```diff
@@ -0,0 +1,23 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React from 'react';
+
+export default function Asterisk() {
```

Comment:
We thought that making it it's own component makes it reusable. If more and more forms will be converted to React the component does make sence to be it's own. If you still think it is better to move it to the `CsvToDatabase.jsx` we can of course do that.

Reply:
Reusability is nice, but this particular component is closely tied to patterns that happen to be convenient with this table layout, which is not a UI that is going to be used long-term.

As more forms are converted to react I would like to build the input components such as `FormInput` and `FormSelect` introduced by this PR into a consistent UI/UX for forms. Forms would be simple to define rather than each form page reimplementing the same state and layout logic. An asterisk would likely end up just being a css class within that framework.